src/reducers/uiReducer.ts

```diff
@@ -31,7 +31,8 @@ const initialState: IUIState = {
   trainingTooltipsShowAll: false,
 };
 
-const uiReducer = (state = initialState, action: any) => {
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.